modules/howtos/pages/managing-connections.adoc

```diff
@@ -230,6 +230,41 @@ INFO: DNS SRV enabled, but less or more than one seed node given.
 Proceeding with normal bootstrap.
 ----
 
+
+== Waiting for Bootstrap Completion
+
+Sometimes, cluster resources are not ready, and a simple connection attempt will result in a timeout error.
```

Comment:
Maybe this can be made a little more clear.. something like "Depending on the environment and network latency, bootstrapping the SDK fully might take a little longer. Since the default Key/Value timeout is only 2.5 seconds, you might see timeouts during bootstrap. To prevent those early timeouts from happening, you can use the waitUntilReady method .... or something like this?